README.md

```diff
@@ -1,16 +1,162 @@
-# Title
+# Return Values
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Identify implicit return values in Ruby syntax
+- Recognize the explicit `return` keyword
 
 ## Introduction
 
-## SWBAT 1
+Displaying text in the console is not the only way available to get information
+from our Ruby methods. While `puts` and `print` display information, they don't
+give a _return value_. When writing code, sometimes we'll want our methods to
+evaluate code and _return_ a value -- not just output text, but return it to the
```

Comment:
please switch to using `&mdash;` here. Github doesn't seem to convert `--` to —

Reply:
could you please doc that in our style guide and update your process as well? thanks

Reply:
Gotcha, thanks!